pkg/unikontainers/unikontainers.go

```diff
@@ -205,7 +209,9 @@ func (u *Unikontainer) Exec() error {
 	}
 
 	// handle network
-	netManager, err := network.NewNetworkManager(u.getNetworkType())
+	networkType := u.getNetworkType()
+	Log.WithField("network type", networkType).Error("network type")
```

Comment:
Maybe it would be better to avoid Error logs for non-error cases. We can use either Debug or Info as a log level.

Reply:
You are right, I have changed the log level.